docs/plugins.md

```diff
@@ -13,25 +13,34 @@ Plugins are automatically loaded if you have them installed in the same `node_mo
 
 When plugins cannot be found automatically, you can load them with:
 
-- The [CLI](cli.md), via the `--plugin` and `--plugin-search-dir`:
+- The [CLI](cli.md), via `--plugin` and `--plugin-search-dir`:
 
   ```bash
-  prettier --write main.foo --plugin-search-dir=./dir-with-plugins --plugin=./foo-plugin
+  prettier --write main.foo --plugin=./foo-plugin --plugin-search-dir=./dir-with-plugins
   ```
 
   > Tip: You can set `--plugin` or `--plugin-search-dir` options multiple times.
 
-- Or the [API](api.md), via the `plugins` and `pluginSearchDirs` options:
+- The [API](api.md), via the `plugins` and `pluginSearchDirs` options:
 
   ```js
   prettier.format("code", {
     parser: "foo",
-    pluginSearchDirs: ["./dir-with-plugins"],
     plugins: ["./foo-plugin"],
+    pluginSearchDirs: ["./dir-with-plugins"],
   });
   ```
 
-Prettier expects each of `pluginSearchDirs` to contain `node_modules` subdirectory, where `@prettier/plugin-*`, `@*/prettier-plugin-*` and `prettier-plugin-*` will be searched. For instance, this can be your project directory or the location of global npm modules.
+- Or the [Configuration File](configuration.md):
+
+  ```json
+  {
+    "plugins": ["./foo-plugin"],
+    "pluginSearchDirs": ["./dir-with-plugins"]
+  }
+  ```
+
+The path that is provided to `pluginSearchDirs` should generally contain a `node_modules` subdirectory. For instance, this can be your project directory or the location of global npm modules. You can also provide a path that is the direct parent directory of your plugins if needed. That is, a path directly to `node_modules` or some arbitrary directory that contains your plugins. Either way, the path you provide is where `prettier-plugin-*`, `@prettier/plugin-*`, and `@*/prettier-plugin-*` will be searched.
```

Comment:
> The path that is provided to `pluginSearchDirs` should generally contain a `node_modules` subdirectory.

I don't think this is true, I used `--plugin-search-dirs=./node_modules` a lot, there is no `node_modules` subdirectory, the `pluginSearchDirs` is the `node_modules`  .

Reply:
Reworded it to be a bit more general.